src/main/scala/viper/gobra/frontend/info/implementation/typing/ghost/GhostExprTyping.scala

```diff
@@ -303,6 +311,8 @@ trait GhostExprTyping extends BaseTyping { this: TypeInfoImpl =>
 
       case PBlankIdentifier() => true
 
+      case PMagicWand(left, right) => Seq(left, right).forall(go)
```

Comment:
Should it be `!strong` because it's a resource in itself?

Reply:
good point